blueocean-plugin/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.jenkins-ci.plugins</groupId>
+        <artifactId>plugin</artifactId>
+        <version>2.30</version>
+    </parent>
+    <name>TODO Amazing Plugin</name>
+    <description>TODO</description>
+    <artifactId>${artifactId}</artifactId>
+    <version>${version}</version>
+    <packaging>hpi</packaging>
+    <url>https://wiki.jenkins-ci.org/display/JENKINS/TODO+Amazing+Plugin</url>
+    <properties>
+        <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
+        <jenkins.version>2.7.4</jenkins.version>
+        <java.level>7</java.level>
+        <node.version>6.2.1</node.version>
+        <npm.version>3.10.10</npm.version>
+        <blueocean.version>1.2.0</blueocean.version>
+        <!-- Other properties you may want to use:
+          ~ jenkins-test-harness.version: Jenkins Test Harness version you use to test the plugin. For Jenkins version >= 1.580.1 use JTH 2.0 or higher.
+          ~ hpi-plugin.version: The HPI Maven Plugin version used by the plugin..
+          ~ stapler-plugin.version: The Stapler Maven plugin version required by the plugin.
+     -->
+    </properties>
+    <!-- The default licence for Jenkins OSS Plugins is MIT. Substitute for the applicable one if needed. -->
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://opensource.org/licenses/MIT</url>
+        </license>
+    </licenses>
+    <dependencies>
+        <dependency>
+            <groupId>io.jenkins.blueocean</groupId>
+            <artifactId>blueocean</artifactId>
```

Comment:
This seems like a terrible dependency (even worse than `workflow-aggregator`, which I recommend people never use). This will pull in BitBucket support, and all kinds of crap you do not need for testing, and which users would not necessarily want to have installed. Declare the minimal dependencies you need to actually compile, and then add `test`-scoped dependencies for the minimal plugins you must install to actually see this run.